supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -131,14 +215,89 @@
 
 *See also*:
 
+// OCP: General; kept as is;
 [discrete]
-[[opex]]
-==== image:images/yes.png[yes] OpEx (noun)
-*Description*: "OpEx" is an abbreviation of "operating expenses".
+[[openshift]]
+==== image:images/yes.png[yes] OpenShift (noun)
+*Description*: The OpenShift product name should be paired with its product distribution or variant name whenever possible. For example:
+
+- OpenShift Container Platform
+- OpenShift Online
+- OpenShift Dedicated
+- OpenShift Kubernetes Engine
+
+Previously, the upstream distribution was called OpenShift Origin, however it is now called OKD; use of the OpenShift Origin name is deprecated.
```

Comment:
Colon needed before "however". Here's a suggestion to simplify the sentence overall.

The [current] name of the upstream distribution is OKD.  The name OpenShift Origin is deprecated.